ORStools/utils/maptools.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, dlg):
         self.moving = None
         self.moved_idxs = 0
         self.error_idxs = 0
-        self.click_dist = 10
+        self.click_dist = 25
```

Comment:
I think this is where a nice commit message might improve things, as someone might be unsure why 25 was chosen here, and the commit message explaining this, along the lines of "we had 10 for testing, was too small, chose 25 as it felt better" would make it clearer that there is no deep reasoning behind the number - and if there is, the commit message could explain *that*.